tests/Client.UiTests/Runner.fs

```diff
@@ -0,0 +1,22 @@
+module Client.UiTests.Runner
+
+open canopy.classic
+open canopy.types
+open Expecto
+
+let startBrowser() =
+    //start Chrome // Use this if you want to see your tests in the browser
+    start ChromeHeadless
+    resize (1280, 960)
+
+[<EntryPoint>]
+let main args =
+    try
+        try
+            startBrowser()
+            runTestsWithArgs { defaultConfig with runInParallel = false } args Tests.tests
```

Comment:
Why not in parallel?

Reply:
I think it is preferrable, so have tests more or less reproducible without introducing any server-side races